Update documentation for 2.6.0 (#484)
ServiceNow/azimuth: docs/docs/user-guide/settings.md

# Settings

The settings page allows the user to edit the config.
Click on :fontawesome-solid-gear: in the top right of the Azimuth app to access it.

![Screenshot](../_static/images/settings.png)

* `Discard` any current modifications if you change your mind.

> * When clicking `Apply and Close`, the start-up tasks may start again, depending on the changes made to the config.

Reviewer: I'm sorry, we changed the "c" to lowercase. You could update the screenshot too. The `Apply and close` button should also be disabled now if there is nothing to apply (before you edit anything).
```suggestion
* When clicking `Apply and close`, the start-up tasks may start again, depending on the changes made to the config.
```

Author: No problem, I think it's fine for the screenshot.

> * Only some fields are modifiable through this screen.

Reviewer: Detail, but I think "editable" is more common than "modifiable". Also, I think "some" -> "select" is perfect here
> adjective (of a group of people or things) carefully chosen from a larger number as being the best or most valuable.
```suggestion
* For now, only select fields are editable through this screen.
```
